packages/cli/src/bin/index.ts

```diff
@@ -107,7 +108,31 @@ const cli = async (
     disable
   };
   try {
-    const outcome = await axeTestUrls(urls, testPageConfigParams, events);
+    let outcome;
+    try {
+      outcome = await axeTestUrls(urls, testPageConfigParams, events);
+    } catch (e) {
+      // Provide a more user-friendly error message when there's a ChromeDriver/Chrome version mismatch.
+      if (
+        e instanceof selenium_error.SessionNotCreatedError &&
+        e.message.includes(
+          'This version of ChromeDriver only supports'
+          // This string has to match the error message printed by chromedriver, see
+          // https://chromium.googlesource.com/chromium/src/+/refs/tags/110.0.5481.194/chrome/test/chromedriver/chrome_launcher.cc#300.
+        )
+      ) {
+        console.error(error('Error: %s'), e.message);
+        console.log(`\nPlease install a matching version of ChromeDriver and run axe with the --chromedriver-path option:
+
+    $ npm install -g chromedriver@<version>
+    $ axe --chromedriver-path $(npm root -g)/chromedriver/bin/chromedriver <url...>
+
+(where <version> is the first number of the "current browser version" reported above.)`);
```

Comment:
"the first number" is a bit ambiguous. We should be very specific about what we want as browser version `114.0.5735.198` the first number would be `1` which would not be the version of chromedriver we want.

Reply:
You're confusing "digit" with "number". I changed your suggestion of "major version" to "first number of the version" since I think there's a good chance that the user isn't familiar with SemVer.